### ISSUE TITLE
geoip2 plugin is working incorrect - no countries

### ISSUE THREAD

**Issue by NickSniper** [Micdu70/geoip2-rutorrent#2]
[Hello!

I have installed the plugin by instruction (phar installed too). The plugin is loaded, enabled. Look at the screenshot [https://drive.google.com/file/d/1_TBWu6C2lwXS-R7QQacrxcgi5-gwbrk1/view](https://drive.google.com/file/d/1_TBWu6C2lwXS-R7QQacrxcgi5-gwbrk1/view) - you can find the column Country, but it is empty inside. 
I have installed it on Qnap NAS, rtorrent 0.9.7/0.13.7

**Comment by Micdu70** [comment]
Hello,

Please check browser console / PHP & Webserver logs.

**Comment by NickSniper** [comment]
Thank you for the quick reply. I've found the error in the PHP log: "FastCGI-stderr: PHP Fatal error:  Uncaught Error: Call to undefined function ctype_digit() in /share/CACHEDEV1_DATA/.qp
kg/Entware-3x/share/www/rutorrent/plugins/geoip2/lookup.php:17"
I have installed the module php7-mod-ctype and restarted the service - it was resolved the issues, now all is working. Thank you.
If I can suggest to you - how about to add php7-mod-ctype to requirements to readme file?
